support-frontend/assets/helpers/tracking/ophan.js

```diff
@@ -81,17 +80,15 @@ type OphanABPayload = {
 
 // ----- Functions ----- //
 
-const trackComponentEvents = (componentEvent: OphanComponentEvent) =>
-  maybeTrack(() => ophan.record({ componentEvent }));
+const trackComponentEvents = (componentEvent: OphanComponentEvent) => ophan.record({ componentEvent });
 
-const pageView = (url: string, referrer: string) =>
-  maybeTrack(() => {
-    try {
-      ophan.sendInitialEvent(url, referrer);
-    } catch (e) {
-      console.log(`Error in Ophan tracking: ${e}`);
-    }
-  });
+const pageView = (url: string, referrer: string) => {
+  try {
+    ophan.sendInitialEvent(url, referrer);
+  } catch (e) {
+    console.log(`Error in Ophan tracking: ${e}`);
```

Comment:
relevant to the current situation but not this PR - presumably this doesn't end up anywhere we can track it (e.g. cloudwatch)